pom.xml

```diff
@@ -102,9 +102,9 @@
         <dependency.postgresql.version>42.2.19</dependency.postgresql.version>
         <dependency.mysql.version>8.0.23</dependency.mysql.version>
         <dependency.mariadb.version>2.7.2</dependency.mariadb.version>
-        <dependency.tas-utility.version>3.0.42</dependency.tas-utility.version>
+        <dependency.tas-utility.version>3.0.43-SNAPSHOT</dependency.tas-utility.version>
```

Comment:
I presume you are going to "release" the TAS library before you merge this i.e. don't merge using a SNAPSHOT artefact.